src/main/java/io/mathdojo/Topic.java

```diff
@@ -0,0 +1,106 @@
+package io.mathdojo;
+
+import java.util.List;
+import java.util.Map;
+
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+@Document(collection = "topics")
+public class Topic {
+	public static Topic EMPTY_DATABASE = new Topic("The-database-is-empty", null, null, null);
+	@Id
+	private String id;
+	private String topicTitle;
+	private String name;
+	private List<String> questions;
+
+	public Topic() {
+
+	}
+
+	public Topic(String id, String title, String name, List<String> questions) {
+		super();
+		this.id = id;
```

Comment:
Probably worth enforcing that ids and titles are the same ( I believe that was our intent)